webapp/Connector/src/store/MabStatus.js

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2018 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+Ext.define('Connector.store.MabStatus', {
+
+    extend: 'Ext.data.Store',
+
+    alias: 'store.mabstatus',
+
+    model: 'Connector.model.MabDetail',
+
+    constructor : function(config) {
+        this.fetchCount = 0;
+        this.loadCount = 0;
+        this.results = {};
+
+        this.callParent([config]);
+    },
+
+    createModel : function(config) {
+        return Ext.create(this.model, Ext.applyIf(config, {
+            activeCountLink: true,
+            modelClass: 'Connector.model.MabPane',
+            subcount: -1,
+            viewClass: 'Connector.view.MabPane'
+        }));
+    },
+
+    executeAll : function(queries) {
+        this.fetchCount++;
+        var fetchCount = this.fetchCount;
+        var queryCount = Object.keys(queries).length;
+
+        for (var q in queries) {
+            if (queries.hasOwnProperty(q)) {
+                LABKEY.Query.executeSql({
+                    schemaName: 'cds',
+                    sql: queries[q],
+                    success: this.onExecute.bind(this, {
+                        fetchCount: fetchCount,
+                        key: q,
+                        queryCount: queryCount
+                    })
+                })
+            }
+        }
+    },
+
+    load : function() {
+        Connector.getState().onReady(function(state) {
+
+            // listen to state for mab filter changes
+            if (!this.filterListener) {
+                this.filterListener = true;
+                state.on('mabfilterchange', this.load.bind(this));
+            }
+
+            Connector.getQueryService().onQueryReady(function() {
+                this.executeAll({
+                    gridBase: MabQueryUtils.getBaseCountSQL(true),
```

Comment:
these queries don't seem to respect logQuery url param

Reply:
Added.